### DISCUSSION TITLE
OCR检测v4server微调后可能出现【BGR通道或者RGB通道的图片】有其一检测不出结果

### DISCUSSION THREAD

**Discussion by Iloccorb96** [PaddlePaddle/PaddleOCR#12326] [Q&A]
如题，微调了3个版本的模型，其中一个版本会出现某张图片检测不出结果，但是将BGR调整为RGB或者反之，就可以正常检测。训练模型、加载模型的参数都是一样的....请问是模型的什么缺陷吗
图片如下，合成的样本....
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/130543358/c01d33a7-d7aa-45e9-aad7-337f8ef845b3)

**Comment by Sunting78** [comment]
您好，OCR检测模型配置文件中的图像读入模式是 img_mode: BGR 只有其一能检测是正常的

**Comment by Iloccorb96** [comment]
可能我没说清楚...同一个模型，有些图片是BGR不行，调整为RGB就可以了。但是另一些图片是RGB不行，调整为BGR就可以了，不清楚模型底层是什么原因...
